### PR TITLE
Feature/add dataset selector

### PR DESCRIPTION
**Pull request by perlo27**

### CODE REVIEW

File: src/utils/dataParser.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export function parseGoogleSheetsApiResponse(
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     const [_, title] = sheet.properties.title.split(':');
     if (!isNaN(Date.parse(sheet.properties.title))) {
-      acc[title] = flattenSheet(sheet.data[0].rowData);
+      acc[title.trimStart()] = flattenSheet(sheet.data[0].rowData);
```
Comment (Aydoun) [inline src/utils/dataParser.ts]: trim() is better concerning browser support!

File: cypress/integration/quadrant.ts [hunk 2]
```diff
@@ -123,12 +126,13 @@ describe('quadrant', () => {
...
 
       it('should select an element in list when i click on item in dropdown, and close dropdown', () => {
-        dataTestId('search-input').type('a');
+        dataTestId('search-input').type('react');
 
         dataTestId('search-technology')
           .first()
           .click()
           .then(el => {
+            console.log('---> el ', el.text());
```
Comment (Aydoun) [inline cypress/integration/quadrant.ts]: no need for this i suppose

File: src/Theme/index.ts [hunk 3]
```diff
@@ -39,6 +39,7 @@ export const baseTheme = {
     warning: '#fcea0b',
     danger: '#ff5100',
     light: '#f3f3f3',
+    borderLight: `#f1f1f1`,
```
Comment (Aydoun) [inline src/Theme/index.ts]: #f1f1f1 and #f3f3f3 are very close, we could reuse light variable

File: src/components/DataSetFilter/index.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+          viewBox="0 0 24 24"
+          aria-hidden="true"
+          role="presentation"
+        >
+          <path d="M7 10l5 5 5-5z" />
+        </Triangle>
+      </Selected>
+      {opened && (
+        <DropDown opened={opened} data-testid="dataSetFilter-dropdown">
+          {availableDates.map(date => (
```
Comment (Aydoun) [inline src/components/DataSetFilter/index.tsx]: couldn't this part be made with the css display property?

File: src/utils/dataParser.ts (hunk 1 above)
Comment (perlo27) [reply src/utils/dataParser.ts]: I assume all the devs in Frontmen have more less modern browser, but i'll fix :)

File: src/Theme/index.ts (hunk 3 above)
Comment (perlo27) [reply src/Theme/index.ts]: make sense

File: src/components/DataSetFilter/index.tsx (hunk 4 above)
Comment (perlo27) [reply src/components/DataSetFilter/index.tsx]: what do you mean?

**Pull request merged**